.github/workflows/actions-pr.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt in the project root for license information.
+
+name: PR Build
+
+on: [ push ]
+
+env:
+  # Path to the project source.
+  PROJECT_PATH: VSConfigFinder
+
+  # Path to the test project source.
+  TEST_PROJECT_PATH: VSConfigFinder.Test
+
+  # Configuration type to build.
+  BUILD_CONFIGURATION: Release
+
+jobs:
+  build:
+    runs-on: windows-2022
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install .NET Core SDK
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 7.x
```

Comment:
Is it possible to specify just to use the global.json version?

Reply:
I'll look into it.

Reply:
Added